### DISCUSSION TITLE
How to create a writable volume

### DISCUSSION THREAD

**Discussion by ghost** [lima-vm/lima#396] [Q&A] [answered]
I was wondering if there is a way to mount a writable volume...

If I run this:
```
docker run -d -p 5432:5432  -v ~/workplace/dbs/my-db:/var/lib/postgresql/data  postgres
```

Postgres is not able to write on /var/lib since it is mounted as readonly volume.
How can I set that volume writable?

**Comment by AkihiroSuda** [comment, marked as answer]
Set this field to true (~/.lima/default/lima.yaml)

https://github.com/lima-vm/lima/blob/e494f7450b6100149bf6adefa9becfbfacbafdad/pkg/limayaml/default.yaml#L40-L44

**Comment by ghost** [comment]
@AkihiroSuda thank you
That "writable" option applies to all the files and folders inside `~`? So each time I create a volume prefixed with `~` as source path it is mounted as readonly? 
Is it safe to set that to true?

**Reply by greghao** [reply]
Not sure if this applies to you but the way I got around this is to simply mount another share and made that writable instead of the whole "~"
